ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md

```diff
@@ -81,3 +82,49 @@ Where:
 * you can check the mapping between partition IDs and partition names using `gdisk -l` command
 
 If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+    - Boot File: boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-XXX.img.gz
+    - Root File: 	linaro-stretch-alip-qcom-snapdragon-arm64-XXX.img.gz
+
+
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+    - Boot File: boot-sdcard-linaro-buster-dragonboard-410c-XXX.img.gz
+    - Root File: linaro-buster-alip-dragonboard-410c-XXX.img.gz
+
+
+  For Example:
+  ```shell
+  $ wget http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/boot-sdcard-linaro-stretch-qcom-snapdragon-arm64-20171016-283.img.gz
```

Comment:
please use wildcards

Reply:
hmmm... seems to be working. looks like the link in the first section are broken and I was thrown away by: " Warning: wildcards not supported in HTTP."
I'll correct the lins